src/database.js

```diff
@@ -0,0 +1,33 @@
+import { MongoClient } from 'mongodb';
+
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+const { DB_USER, DB_PASSWORD } = process.env;
```

Comment:
create a `constants.js` file and import all the env variables from there. and set default also.

Reply:
On it